server.go

```diff
@@ -63,10 +66,25 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			s.Logger.Printf("Errored %v", err)
 		}
 
+		layoutHtml := []byte("")
+		contentHtml := []byte("")
+		isLayout := true
+
 		for _, result := range results {
 			s.Logger.Printf("Fetched %s in %v", result.Url, result.Duration)
-			w.Write(result.Body)
+
+			if isLayout {
+				layoutHtml = result.Body
+			} else {
+				contentHtml = append(contentHtml, result.Body...)
+			}
+
+			// Assume first result is the layout
+			isLayout = false
```

Comment:
```suggestion
				contentHtml = append(contentHtml, result.Body...)
```

I think this will work paired with the two suggestions above.